x/vesting/keeper/msg_server.go

```diff
@@ -68,6 +70,13 @@ func (k Keeper) CreateClawbackVestingAccount(
 		)
 	}
 
+	// Check for contract account (code hash is not empty)
+	if ethAcc.CodeHash != crypto.Keccak256Hash([]byte{}).String() {
```

Comment:
this would be more straight forward I think?
```suggestion
	if len(ethAcc.CodeHash) != 0 {
```

Reply:
I think that the Keccak256 does not map empty bytes to the empty string.

Reply:
@MalteHerrmann there is a default hash for the zero value so it's length won't be zero